example/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++")
+
+add_executable (hello hello.c)
+set_source_files_properties(hello.c PROPERTIES LANGUAGE CXX)
```

Comment:
Would this work with LANGUAGE C?

Reply:
No. With `LANGUAGE C` `CMake` uses `clang` for linking and is then unable to find symbols:
```
hello.c:(.text+0x197): undefined reference to `wasm_i32_val'
...
```

Reply:
Okay, I just merged #3, which should fix this issue. Should compile with C now.

Reply:
(You stlll need to link with clang++, though, or include the c++ stdlib manually.

Reply:
Great. Just tried it out and it works.